File: app/src/main/java/com/raise/practice/MainActivity.java
package com.raise.practice;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

/**
 * Created by raise.yang on 20/06/19.
 */

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView tv = new TextView(this);
        tv.setText("你好，世界");
        setContentView(tv);
    }
}

File: ali_tee/src/main/java/com/abupdate/ali_tee/MainActivity.kt
package com.abupdate.ali_tee

import android.annotation.SuppressLint
import android.content.Context
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.abupdate.common.CommonVal
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    companion object {
        @SuppressLint("StaticFieldLeak")
        lateinit var CTX: Context
    }

    init {
        CommonVal.setContext(this)
        CTX = this
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        layout_test.loadTest(TestHelper(layout_test))
    }
}
